src/role-access/containers/remove-role/remove-role.component.ts

```diff
@@ -0,0 +1,68 @@
+import { Component, OnInit } from '@angular/core';
+import { ActivatedRoute, Router } from '@angular/router';
+import { CaseRole, RemoveAllocationNavigationEvent } from '../../../role-access/models/case-role.interface';
+import { Answer } from '../../../role-access/models';
+import { catchError } from 'rxjs/operators';
+import { handleFatalErrors, WILDCARD_SERVICE_DOWN } from '../../../work-allocation-2/utils';
+import { EMPTY } from 'rxjs';
+import { AllocateRoleService } from '../../../role-access/services/allocate-role.service';
+
+@Component({
+  selector: 'exui-remove-role',
+  templateUrl: './remove-role.component.html'
+})
+export class RemoveRoleComponent implements OnInit {
+
+  public removeAllocationNavigationEvent = RemoveAllocationNavigationEvent;
+  public answers: Answer[] = [];
+  public caption = null;
+  public heading = 'Remove allocation';
+  public hint = 'This will remove the role allocation. You may need to unassign or reassign associated tasks too.';
+  public infoMessage = 'You\'ve removed a role allocation. You may need to unassign or reassign associated tasks too.';
```

Comment:
Can these move to answer-text.ts like the delete exclusion does

Reply:
Done